api/app/signals/apps/my_signals/README.md

```diff
@@ -2,13 +2,19 @@
 
 _Currently, this app is in active development_  
 
-This app will return the signals created in the last 12 months by the logged in reporter.
+This app provides endpoints that will make it possible for reporters to request a login link and view all signals created in the past 12 months.
+
+Flow:
+* A login link should be requested
+  * If a reporter has created any Signal in the last 12 months an email will be sent containing a login link
+* With the login link all Signals created in the last 12 months can be retrieved
```

Comment:
signals of the current logged in user right

Reply:
Correct, updated the sentence.